### PR TITLE
Jackal nav

### PR DESCRIPTION
**Pull request by shokoofeh**
Modified the Wheel separation multiplier to fix the difference between cmd_vel and odometry reading.

### CODE REVIEW

File: jackal_navigation/config/local_costmap_params.yaml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 local_costmap:
-   global_frame: /odom
-   robot_base_frame: /base_link
+   global_frame: map
```
Comment (mikepurvis) [inline jackal_navigation/config/local_costmap_params.yaml:2]: This seems like an odd change. Are you sure?

File: jackal_navigation/config/odom_nav_params/global_costmap_params.yaml [hunk 2]
```diff
@@ -2,19 +2,12 @@ global_costmap:
...
-     name: inflater
-     type: "costmap_2d::InflationLayer"
+  publish_frequency: 5
+   width: 40.0
+   height: 40.0
+   resolution: 0.05
+   origin_x: -20.0
+   origin_y: -20.0
+   static_map: true
+   rolling_window: false
```
Comment (mikepurvis) [inline jackal_navigation/config/odom_nav_params/global_costmap_params.yaml]: Do we not like the layer plugins, or is this just a case of accepting the defaults?

Also, please remove the three-space indent (same in local_costmap_params.yaml).

File: jackal_navigation/launch/odom_navigation_demo.launch [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
+        <rosparam file="$(find jackal_navigation)/config/base_local_planner_params.yaml" command="load" />
+        <rosparam file="$(find jackal_navigation)/config/move_base_params.yaml" command="load" />
+
+        <param name="base_global_planner" type="string" value="navfn/NavfnROS" />
+        <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>
+        <rosparam>
+            base_local_planner/TrajectoryPlannerROS:
+                xy_goal_tolerance: 0.15
+                yaw_goal_tolerance: 0.314
+        </rosparam>
```
Comment (mikepurvis) [inline jackal_navigation/launch/odom_navigation_demo.launch]: Is this an override of `base_local_planner_params.yaml`? Please document rationale, or move this into the yaml file.

File: jackal_navigation/launch/odom_navigation_demo.launch [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
+
+        <remap from="odom" to="odometry/filtered" />
+    </node>
```
Comment (mikepurvis) [inline jackal_navigation/launch/odom_navigation_demo.launch]: Please revert this to 2-space indent, and ensure that the files end with a newline.

File: jackal_navigation/maps/8.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+image: map.pgm
+resolution: 0.020000
+origin: [-38.560000, -30.240000, 0.000000]
+negate: 0
+occupied_thresh: 0.65
+free_thresh: 0.196
+
```
Comment (mikepurvis) [inline jackal_navigation/maps/8.yaml]: Does this map correspond to a demo world we're planning to make available in `jackal_gazebo`?

File: jackal_navigation/config/local_costmap_params.yaml (hunk 1 above)
Comment (shokoofeh) [inline jackal_navigation/config/local_costmap_params.yaml:2]: Yes. /odom is the world frame of the odometry navigation and /map is the world frame of the localization.

File: jackal_navigation/config/odom_nav_params/global_costmap_params.yaml (hunk 2 above)
Comment (shokoofeh) [inline jackal_navigation/config/odom_nav_params/global_costmap_params.yaml]: I added these plugins to the common parameters for both glocal and local costmaps so they are removed from global costmap params and added to the common costmap params.

File: jackal_navigation/maps/8.yaml (hunk 5 above)
Comment (shokoofeh) [inline jackal_navigation/maps/8.yaml]: Yes, the world (jackal_race.world) is added to the worlds of jackal_gazebo. For accessing this world, we need to pass the "jackal_race.world" to the "world_name" arg.

Comment (mikepurvis) [inline jackal_navigation/maps/8.yaml]: Okay, cool. Please change the name of this to match that, and ensure that all of this is documented on the following pages as appropriate: http://wiki.ros.org/jackal_simulator, http://wiki.ros.org/jackal_navigation

**Comment by shokoofeh** [comment]
@mikepurvis I applied your reviews at f083e1fe7821a193b717813504bf3cb722309441

**Comment by mikepurvis** [comment]
Sweet, thanks @Shokoofeh!

**Pull request merged**